src/libsystemd/sd-bus/bus-common-errors.h

```diff
@@ -81,6 +81,8 @@
 #define BUS_ERROR_TRANSFER_IN_PROGRESS         "org.freedesktop.import1.TransferInProgress"
 
 #define BUS_ERROR_NO_PRODUCT_UUID              "org.freedesktop.hostname1.NoProductUUID"
+#define BUS_ERROR_NO_PRIVILEGE_TO_UPDATE_FILE  "org.freedesktop.hostname1.NoPrivilegeToUpdateFile"
+#define BUS_ERROR_READONLY_FILESYSTEM          "org.freedesktop.hostname1.ReadOnlyFilesystem"
```

Comment:
hmm, looking at this closer: translating CamelCase to UPPER_CASE would mean ReadOnlyFilesystem would map to READ_ONLY_FILESYSTEM.  or the otherway round: READONLY_FILESYSTEM would map to ReadonlyFilesystem. 

I'd probably stick to READ_ONLY_FILE_SYSTEM and ReadOnlyFileSystem though